### PR TITLE
Update to v1.9.4

### PR DESCRIPTION
**Pull request by zioproto**
v1.9.4 is required to use Virtual Machines with IPv6 addresses.
See https://github.com/kubernetes/kubernetes/pull/59868 for details.

### CODE REVIEW

**Comment by zioproto** [comment]
Not sure why the test fails with `rake` not found. Is the gate broken ?

**Comment by ctrlaltdel** [comment]
Yes, it seems that travis tries to use some default settings for ruby project when no `.travis.yml` file is present in the repo. This will be fixed as soon as #13 is merged :)

File: README.md [hunk 1]
```diff
@@ -51,11 +51,13 @@ The following environment variables needs to be defined:
...
   * `OS_PASSWORD`
   * `OS_USERNAME`
+  * `OS_DOMAIN_NAME`
 
-Author
+Authors
 ------
 
   * François Deppierraz <francois.deppierraz@infraly.ch>
+  * Saverio Proto <saverio.proto@switch.ch>
```
Comment (ctrlaltdel) [inline README.md]: Already added in https://github.com/infraly/k8s-on-openstack/commit/241d9671baff25ef3a2cb4ba5984586faa89b2fc

**Comment by ctrlaltdel** [comment]
I'm going to close this PR in favor of #16, because I haven't (yet) found a way to get travis-ci to test this PR with the secure environment variables required (OpenStack credentials).

**Pull request closed without merging**